lib/rules/assert-args.js

```diff
@@ -48,13 +48,13 @@ module.exports = {
         function getAssertContext() {
             assert.ok(testStack.length);
 
-            return testStack[testStack.length - 1].assertContextVar;
+            return testStack.at(-1).assertContextVar;
```

Comment:
[array#at](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) requires Node 16.6. So I might wait to make this change until Node 16 support is dropped in [September](https://nodejs.dev/en/about/releases/) for this. What do you think?

Reply:
Interesting that eslint-plugin-unicorn only requires Node 16 (with their v47) but they put that rule in their recommended config 😕 I would be okay with either us changing our engines requirement to 16.6+ or disabling that rule for now. Do you have a strong preference?

Reply:
Personally, I would default to disabling the rule for now. However, I have no strong preference, so I'm ready to approve whichever route you choose.

Reply:
Hopefully resolved with the latest commit.